apps/dashboard/app/apps/nav_bar.rb

```diff
@@ -81,6 +85,15 @@ def self.nav_profile(item, category, subcategory)
     OodAppLink.new(profile_data).categorize(category: category, subcategory: subcategory)
   end
 
+  def self.nav_page(item, category, subcategory)
+    page_code = item.fetch(:page)
+    page_data = item.clone
```

Comment:
I think it's fairly common to just chain all this is in one line. I believe the `to_h` cast will catch nil.

```suggestion
    page_code = item.fetch(:page).to_h.clone
```